arthur_bench/scoring/scorer.py

```diff
@@ -55,9 +66,9 @@ def run_batch(
         reference_batch: Optional[List[str]] = None,
         input_text_batch: Optional[List[str]] = None,
         context_batch: Optional[List[str]] = None,
-    ) -> List[float]:
+    ) -> List[Feedback]:
```

Comment:
This doesn't match the spec in the design doc. We need to support List[float] for backwards compatibility